Procfile

```diff
@@ -0,0 +1 @@
+web: cd lib/app; bundle exec rackup config.ru -p ${PORT}
```

Comment:
I don't know of a clever way to push this into `Makefile` off the top of my head. It didn't feel important enough to spend more time on.

Reply:
This is the only thing that feels weird to me in this PR. Could this not call `make run` instead? Maybe that directive in the Makefile would need to have a `-p ${PORT:"9292"}`?

Reply:
In the makefile you can write at the top something like:

```
PORT ?= 9292
```
and then 

```
cd lib/app; bundle exec rackup config.ru -p $(PORT)
```

then we you use it, it'll use `9292` or the env variable:

```
PORT=8888 make run
```

Reply:
But then we ignore the port setting in `config/connectors.yml`. It requires parsing YAML to know that value. I couldn't get that working smoothly in Heroku and gave up after ~15mins (`yq` is not available).

Reply:
Had to rebase with _main_. I believe this is the only real concern remaining. Thoughts on whether it's worth resolving or leaving as is?

Reply:
We can always fix it later.